ovos_bus_client/util/scheduler.py

```diff
@@ -330,6 +329,7 @@ def shutdown(self):
         Stop the running thread.
         """
         self._stopping.set()
+        self._running.clear()
```

Comment:
Might this be safer after the shutdown is finished?

Reply:
Makes sense